liferay-echo/liferay-portal
# LPS-195043 Migrate SearchBar.soy to React

## How to test 🧪
- Create two sites with Speedwell and Minium initializers
- Go to Home page of both sites and check the top search bar

![image](https://github.com/liferay-echo/liferay-portal/assets/2444936/136edd66-6d04-43c1-be74-3c414eb55790)

![image](https://github.com/liferay-echo/liferay-portal/assets/2444936/2b53efbf-54e5-4d9b-b8be-a775b289a260)
